src/__swaps__/screens/Swap/components/GasPanel.tsx

```diff
@@ -58,14 +58,7 @@ function UnmountWhenGasPanelIsClosed({ placeholder, children }: PropsWithChildre
 
 function PressableLabel({ onPress, children }: PropsWithChildren<{ onPress: VoidFunction }>) {
   return (
-    <Box
-      as={ButtonPressAnimation}
-      paddingVertical="8px"
-      marginVertical="-8px"
-      onPress={onPress}
-      backgroundColor="accent"
-      style={{ maxWidth: 175 }}
-    >
+    <Box as={ButtonPressAnimation} paddingVertical="8px" marginVertical="-8px" onPress={onPress} style={{ maxWidth: 175 }}>
```

Comment:
removed background color here. I think this was a bug. caused the gas options to have a blue squared background on Android.